pkg/koordlet/resmanager/resctrl_reconcile.go

```diff
@@ -43,6 +44,16 @@ const (
 	BEResctrlGroup = "BE"
 	// UnknownResctrlGroup is the resctrl group which is unknown to reconcile
 	UnknownResctrlGroup = "Unknown"
+
+	// Max memory bandwidth for AMD CPU, Gb/s, since the extreme limit is hard to reach, we set a discount by 0.8
+	// TODO The max memory bandwidth varies across SKU, so koordlet should be aware of the maximum automatically,
+	// or support an configuration list.
+	// Currently, the value is measured on "AMD EPYC(TM) MILAN"
+
+	AMDCCDMaxMBGbps = 25 * 8 * 0.8
```

Comment:
How to support if there are different CPUs?

Reply:
I have set a TODO here, for other AMD CPU models, max memory bandwidth varies across SKU, so koordlet should be aware of the maximum automatically, or support a configuration list according to offline measurement.